README.md

```diff
@@ -42,6 +54,21 @@ Claudie has its own managed load-balancing solution, which you can use for Ingre
 
 Claudie comes pre-configured with a storage solution, with ready-to-use Storage Classes. See [Storage docs](https://github.com/Berops/claudie/tree/master/docs/storage) to learn more.
 
+### Supported cloud providers
+
+- [AWS](./docs/input-manifest/providers/aws.md)
+- [Azure](./docs/input-manifest/providers/azure.md)
+- [GCP](./docs/input-manifest/providers/gcp.md)
+- [OCI](./docs/input-manifest/providers/oci.md)
+- [Hetzner](./docs/input-manifest/providers/hetzner.md)
+
+Adding a support for another provider is easy. Let us know your needs.
+
+### Supported DNS providers
```

Comment:
We actually support DNS on all providers + Cloudflare. So this list should contain
- AWS
- Azure
- GCP
- OCI
- HetznerDNS
- Cloudflare